index.js

```diff
@@ -153,6 +166,11 @@ parser.getFullestName = function(str){
     return name;
 };
 
+/**
+ * Parse an address into address, city, state, zip
+ * @param {string} str - The string we want to parse into address, city, state, zip
+ * @returns {Object} An object containing address, city, state, zip
+ */
```

Comment:
I'm of the opinion that the candidate should be able to read the code and discuss with the interviewer to understand the behavior. please remove doc comments.

Reply:
I've removed the doc comments